mods/ts/mod.yaml

```diff
@@ -266,19 +267,29 @@ ModContent:
 	QuickDownload: basefiles
```

Comment:
You need to package a new quickinstall zip that contains all the essential content (base + firestorm, without music), and then reference it here.